lms/templates/instructor/edx_ace/enrollenrolled/email/body.html

```diff
@@ -7,32 +7,66 @@
-                {% endfilter %}
             </h1>
 
             <p style="color: rgba(0,0,0,.75);">
-                {% filter force_escape %}
-                {% blocktrans %}You have been enrolled in {{ course_name }} at {{ site_name }} by a member of the course staff. This course will now appear on your {{ site_name }} dashboard.{% endblocktrans %}
-                {% endfilter %}
                 <br />
             </p>
 
-            {% filter force_escape %}
-                {% blocktrans asvar course_cta_text %}Access the Course Materials Now{% endblocktrans %}
-            {% endfilter %}
-            {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text course_cta_url=course_url %}
-
             <p style="color: rgba(0,0,0,.75);">
-                {% filter force_escape %}
                 {% blocktrans %}This email was automatically sent from {{ site_name }} to {{ full_name }}{% endblocktrans %}
-                {% endfilter %}
                 <br />
             </p>
         </td>
     </tr>
+
+   <tr>
+        <td style="width: 100%;
+            max-width: 800px;
+            background-color: #FFF;
+            color: #000000;
+            margin: auto;
+            text-align: center;
+            border: 1px solid #F3F3F3;">
+
+<img src="{% static 'pearson-pols-theme/images/PearsonLogo_Primary_Blk_RGB.png' %}" style="width: 150px; padding: 5px;" />
+
+            <div style="width: 90%;
+                        background-color: #D4EAE4;
+                        text-align: left;
+                        padding: 2% 5%;
+                        background-image: url('{% static 'pearson-pols-theme/images/laptop_transparent-bg.png' %}');
+                        background-size: 50%;
+                        background-position: right bottom;
+                        background-repeat: no-repeat;">
+
+                <h1 style="font-family:'Playfair Display', serif; font-weight: 700;">
+                     You have been enrolled in {{ course_name }}
+                </h1>
+                <p style="padding: 0px;
+                          width: 50%;
+                          min-width: 300px;">
+                    {% blocktrans %}You have been enrolled in {{ course_name }} at {{ site_name }} by a member of the course staff. This course will now appear on your {{ site_name }} dashboard.{% endblocktrans %}
+                    <br />
+                </p>
+
+           {% trans "Access the Course Materials Now" as course_cta_text %}
+            {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text course_cta_url=course_url %}
+
+            </div>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <p>
+                 {% blocktrans %}This email was automatically sent from {{ site_name }} to {{ full_name }}{% endblocktrans %}
+                <br />
+            </p>
+        </td>
+    </tr>
+
```

Comment:
It seems like this blank line is not required.

Reply:
There are many details in this template, this template is taken from the existing one, it's not the purpose of this PR to fix them, but to fix the main issue, however, applied.